angular/core/components/thread_page/undecided_panel/undecided_panel.coffee

```diff
@@ -15,3 +15,7 @@ angular.module('loomioApp').directive 'undecidedPanel', ->
 
     $scope.hideUndecided = ->
       $scope.undecidedPanelOpen = false
+
+    $scope.remindUndecided = ->
+      ScrollService.scrollTo('.comment-form__comment-field')
```

Comment:
I'd suggest the scroll should happen inside the comment_from `$on`, since we'll want to be scrolling there anytime we manually manipulate the comment text